response.py

```diff
@@ -63,7 +66,25 @@ def get_instruction(category: str, model: Model, source_lang: str,
 
 
 def get_responses(prompt: str, category: str, source_lang: str,
-                  target_lang: str):
+                  target_lang: str, request: gr.Request):
+  cookie = cookies.SimpleCookie()
+  cookie.load(request.headers["cookie"])
+  token = cookie["token"].value if "token" in cookie else None
+
+  try:
+    rate_limiter.request_allowed(token)
```

Comment:
seems like it increase rate limiter cont. Isn't it better to move this check below category check?

Reply:
you're right. thank you!